nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -18,6 +13,16 @@
     [2.24](https://nix.dev/manual/nix/latest/release-notes/rl-2.24).
   Notable changes include improvements to Git fetching, documentation comment support in `nix-repl> :doc`, as well as many quality of life additions.
 
+- There have been significant changes and improvements to macOS support.
```

Comment:
```suggestion
- There have been significant changes to macOS support.
```

Not sure this part makes sense here as only one of these points is an improvement

Reply:
Honestly the 11.3 bump is very much an improvement on multiple levels, but done :)

Reply:
An argument could be made that making the toolchain behave more like a native toolchain is an improvement in its own right separate from the addition of the extra SDKs.